appeals/e2e/cypress/fixtures/appealsApiRequests.js

```diff
@@ -85,21 +72,75 @@ const appealsApiRequests = {
 			],
 			affectedListedBuildingNumbers: ['10001', '10002'],
 			lpaCostsAppliedFor: false
+		}
+	}
+};
+
+const documentsApiRequest = {
```

Comment:
@RHARV018 I've updated to keep separate documents definition for the type of submission, on top of environment-specific...